.github/workflows/test.yml

```diff
@@ -73,10 +73,17 @@ jobs:
       - name: "Common Test Setup Actions"
         uses: ./.github/actions/common-pre
 
+
+      - name: Setup cmake (macos-latest, wasmer_wamr)
```

Comment:
Had to install an 0.3.x version of cmake which I didn't figure out how to do with `brew` so I added this action here.